doc/users/next_whats_new/2019-06-28-AL.rst

```diff
@@ -0,0 +1,13 @@
+`.Colormap.set_extremes` and `.Colormap.with_extremes`
```

Comment:
I think we've decided not to put links in section titles for aestethic reasons.

But I'm going to merge as is. I'll anyway do a small colormap-related cleanup PR and will update there. While that's a bit of a messy way of working it saves us an extra review roundtrip here.